Admin/AdminBuilder.php

```diff
@@ -22,7 +22,7 @@
 /**
  * @author Christian Gripp <mail@core23.de>
  */
-final class AdminBuilder implements FormBuilderInterface, DatagridBuilderInterface
+final class AdminBuilder implements AdminBuilderInterface
```

Comment:
Cannot mock final classes, so I created an interface to mock. Could be useful if someone wants to create another adminbuilder, but not sure if this is the best solution